browser/app/js/components/Login.js

```diff
@@ -62,6 +62,17 @@ export default class Login extends React.Component {
       type: 'danger',
       message: ''
     }))
+
+    // If we're on an insecure connection, warn the user.
+    if (window.location.protocol === 'http:') {
+      dispatch(actions.showAlert({
+        type: 'danger',
+        message: 'You are on an insecure connection. Your access key and secret '
+               + 'key should be considered exposed to the network upon login. '
+               + 'Please configure certificates for HTTPS access.'
+      }))
```

Comment:
Is that more accurate? Right now, the text boxes on the login page have "access key" and "secret key" as their placeholder text. It might be confusing.